web-app/package.json

```diff
@@ -3,27 +3,30 @@
   "description": "Pub Golf Web App",
   "version": "0.0.1",
   "scripts": {
+    "tailwind": "npx tailwind build static/tailwind.config.css -o static/tailwind.css",
     "dev": "sapper dev",
     "build": "sapper build",
     "export": "sapper export",
     "start": "node __sapper__/build",
     "cy:run": "cypress run",
     "cy:open": "cypress open",
-    "test": "run-p --race dev cy:run"
+    "test": "run-p --race dev cy:run",
+    "postinstall": "run-s tailwind"
   },
   "dependencies": {
     "compression": "^1.7.1",
+    "google-protobuf": "^3.6.1",
+    "grpc-web": "^1.0.0",
```

Comment:
I think these got rearranged automatically when I used `npm install`

Reply:
My bad...I just tacked them on to the end instead of adding them in alphabetical order.